src/random.jl

```diff
@@ -0,0 +1,80 @@
+#=
+Generate random NormalFormGame instances.
+=#
+
+#
+# Random Games Generating
+#
+"""
+    random_game{N}(nums_actions::NTuple{N,Int})
+
+Return a random NormalFormGame instance where the payoffs are drawn
+independently from the uniform distribution on [0, 1).
+
+# Arguements
+* `nums_actions::NTuple{N,Int}`: Tuple of the numbers of actions,
+    one for each player.
+
+# Returns
+* `::NormalFormGame`: The generated N-players random NormalFormGame.
+"""
+function random_game{N}(nums_actions::NTuple{N,Int})
+    if N == 0
+        throw(ArgumentError("nums_actions must be non-empty"))
+    end
+
+    players::NTuple{N,Player{N,Float64}} =
+        ntuple(i -> Player(rand(tuple(nums_actions[i:end]...,
+                                      nums_actions[1:i-1]...))),
+               N)
+
+    return NormalFormGame(players)
+end
+
+#
+# Covariance Games Generating
+#
+"""
+    covariance_game{N, T<:Real}(nums_actions::NTuple{N,Int}, rho::T)
+
+Return a random NormalFormGame instance where the payoff profiles
+are drawn independently from the standard multi-normal with the
+covariance of any pair of payoffs equal to `rho`, as studied in
+[1]_.
```

Comment:
The output is the same as 

```
# References

[^1]: 

    Y. Rinott and M. Scarsini, "On the Number of Pure Strategy
    Nash Equilibria in Random Games," Games and Economic Behavior
    (2000), 274-293.
```

like this:

<img width="769" alt="2017-03-21 3 53 19" src="https://cloud.githubusercontent.com/assets/22466278/24135715/b8adfce8-0e4e-11e7-8a93-1eb0a0aba28b.png">

I think we may have talked about this in the meeting and you said you don't like that redundant line?